### PR TITLE
Guidance for messages design

### PR DESCRIPTION
**Pull request by weralabaj**
Connects to https://github.com/Particular/docs.particular.net/issues/517

### CODE REVIEW

File: nservicebus/messaging/messages-events-commands.md [hunk 1]
```diff
@@ -78,3 +79,13 @@ public class MyEvent : IEvent { }
 
 public interface MyEvent : IEvent { }
 ```
+
+
+ ### Conventions
+
+ Conventions are described in the dedicated [Conventions](/nservicebus/messaging/conventions.md) article.
```
Comment (SimonCropp) [inline nservicebus/messaging/messages-events-commands.md]: should this be a related article in the header?

File: nservicebus/messaging/messages-events-commands.md [hunk 2]
```diff
@@ -78,3 +79,13 @@ public class MyEvent : IEvent { }
...
+
+
+ ## Designing Messages
+
+ Messages represent data contract between the endpoints. When creating messages, one should follow the general guidelines for designing contracts, such as the messages should be as small as possible, they should be simple POCO objects, they should follow a Single Responsibility Principle, etc. Classes used for other purposes (e.g. domain objects, data access objects) should not be used as messages.
```
Comment (SimonCropp) [inline nservicebus/messaging/messages-events-commands.md]: "general guidelines for designing contracts"  this is stated like it is a well known approach. when it isnt really. can we link somewhere?

Comment (SimonCropp) [inline nservicebus/messaging/messages-events-commands.md]: "Single Responsibility Principle" link

Comment (SimonCropp) [inline nservicebus/messaging/messages-events-commands.md]: i think this should be point form

File: nservicebus/messaging/messages-events-commands.md (hunk 1 above)
Comment (weralabaj) [inline nservicebus/messaging/messages-events-commands.md]: I'll add that.

File: nservicebus/messaging/messages-events-commands.md (hunk 2 above)
Comment (weralabaj) [inline nservicebus/messaging/messages-events-commands.md]: > "general guidelines for designing contracts" this is stated like it is a well known approach. when it isnt really. can we link somewhere?

I'll try to look something up

File: nservicebus/messaging/messages-events-commands.md (hunk 1 above)
Comment (weralabaj) [inline nservicebus/messaging/messages-events-commands.md]: already there ;)

**Comment by weralabaj** [comment]
@SimonCropp better now?

**Pull request merged**